packages/generate/src/edgeql-js.ts

```diff
@@ -306,12 +299,9 @@ project to exclude these files.`
   } else if (options.updateIgnoreFile) {
     const gitIgnorePath = path.join(root, ".gitignore");
 
-    let gitIgnoreFile: string | null = null;
-    try {
-      gitIgnoreFile = await readFileUtf8(gitIgnorePath);
-    } catch (err) {
-      console.error("Error occurred while reading file", gitIgnorePath, err);
-    }
+    const gitIgnoreFile = await readFileUtf8(gitIgnorePath)
+      .then((content) => content)
+      .catch(() => "");
```

Comment:
I think this should `catch` to `null` instead of the empty string?

```suggestion
    const gitIgnoreFile = await readFileUtf8(gitIgnorePath)
      .then((content) => content)
      .catch(() => null);
```

Reply:
oh yes, good catch!